core/tests/test_utils.py

```diff
@@ -2379,13 +2407,18 @@ def _get_sent_email_messages(self, to):
             to: str. The recipient email address.
 
         Returns:
-            list(EmailMessageMock). Returns the list of email messages
-            corresponding to that recipient email.
+            list(EmailMessageMock). The list of email messages corresponding to
+            that recipient email.
         """
         return self.emails_dict[to] if to in self.emails_dict else []
 
     def _get_all_sent_email_messages(self):
-        """Gets the entire messages dictionary."""
+        """Gets the entire messages dictionary.
+
+        Returns:
+            dict(list(EmailMessageMock)). The dictionary containing all lists of
```

Comment:
This is not how we write docstrings for dicts (and the typeinfo is not correct either). Please look at other examples in the codebase and make this description clearer.

Reply:
Fixed.